dags/taar_amodump.py

```diff
@@ -1,83 +1,122 @@
+    gcp_conn_id=gcp_conn_id,
+    project_id=connection.project_id,
+    location="us-central1-a",
+    cluster_name="bq-load-gke-1",
+    name="taar-amowhitelist",
+    namespace="default",
+    image="gcr.io/moz-fx-data-airflow-prod-88e0/taar_gcp_etl:0.1",
     owner="vng@mozilla.com",
-    email=["mlopatka@mozilla.com", "vng@mozilla.com"],
-    env=mozetl_envvar("taar_amowhitelist",
-                      {},
-                      {'MOZETL_SUBMISSION_METHOD': 'spark'}),
-    uri="https://raw.githubusercontent.com/mozilla/python_mozetl/master/bin/mozetl-submit.sh",
-    output_visibility="private",
-    dag=dag
+    email=["mlopatka@mozilla.com", "vng@mozilla.com", "hwoo@mozilla.com"],
+    arguments=["-m", "taar_etl.taar_amowhitelist"],
+    env_vars={
+        "AWS_ACCESS_KEY_ID": aws_access_key,
+        "AWS_SECRET_ACCESS_KEY": aws_secret_key,
+    },
+    dag=dag,
 )
 
-editorial_whitelist = EMRSparkOperator(
+editorial_whitelist = GKEPodOperator(
     task_id="taar_update_whitelist",
-    job_name="Generate a JSON blob from editorial reviewed addons for TAAR",
-    execution_timeout=timedelta(hours=1),
-    instance_count=1,
+    project_id=connection.project_id,
+    location="us-central1-a",
+    cluster_name="bq-load-gke-1",
+    name="taar-update-whitelist",
+    namespace="default",
+    image="gcr.io/moz-fx-data-airflow-prod-88e0/taar_gcp_etl:0.1",
     owner="vng@mozilla.com",
-    email=["mlopatka@mozilla.com", "vng@mozilla.com"],
-    env=mozetl_envvar("taar_update_whitelist",
-                      {"date": "{{ ds_nodash }}"},
-                      {'MOZETL_SUBMISSION_METHOD': 'spark'}),
-    uri="https://raw.githubusercontent.com/mozilla/python_mozetl/master/bin/mozetl-submit.sh",
-    output_visibility="private",
-    dag=dag
+    email=["mlopatka@mozilla.com", "vng@mozilla.com", "hwoo@mozilla.com"],
+    arguments=["-m", "taar_etl.taar_update_whitelist", "--date", "{{ ds_nodash }}"],
+    env_vars={
+        "AWS_ACCESS_KEY_ID": aws_access_key,
+        "AWS_SECRET_ACCESS_KEY": aws_secret_key,
+    },
+    dag=dag,
 )
 
-taar_lite = EMRSparkOperator(
+
+taar_lite = SubDagOperator(
     task_id="taar_lite",
-    job_name="Generate GUID coinstallation JSON for TAAR",
-    instance_count=5,
-    execution_timeout=timedelta(hours=4),
-    owner="mlopatka@mozilla.com",
-    email=["mlopatka@mozilla.com", "vng@mozilla.com"],
-    env=mozetl_envvar("taar_lite",
-                      {"date": "{{ ds_nodash }}"},
-                      {'MOZETL_SUBMISSION_METHOD': 'spark'}),
-    uri="https://raw.githubusercontent.com/mozilla/python_mozetl/master/bin/mozetl-submit.sh",
-    output_visibility="private",
-    dag=dag
+    subdag=moz_dataproc_pyspark_runner(
+        parent_dag_name="taar_amodump",
+        dag_name="taar_lite",
+        default_args=default_args,
+        cluster_name=taarlite_cluster_name,
+        job_name="TAAR_Lite_GUID_GUID",
+        python_driver_code="gs://temp-hwoo-removemelater/taar_lite_guidguid.py",
```

Comment:
This will need to change prior to merge

Reply:
@haroldwoo Can you help me get this integrated into CircleCI so that tagged builds automatically push to a GCS bucket?

Reply:
as long as the script doesnt have any secrets, you can put a copy of the script in this repo under the jobs folder, and itll be sync'd to gs://moz-fx-data-prod-airflow-dataproc-artifacts/jobs after this PR merges.

https://github.com/mozilla/telemetry-airflow/blob/master/.circleci/config.yml#L74

Reply:
Thanks - I've put a copy of the script into the data directory in 09d9487